src/overlays/actors/ovl_En_Kbt/z_en_kbt.c

```diff
@@ -82,27 +99,48 @@ s32 func_80B33E8C(PlayState* play) {
     return false;
 }
 
-void func_80B33EF0(EnKbt* this, s16 arg1) {
-    static AnimationHeader* sAnimations[] = {
-        &object_kbt_Anim_000670, &object_kbt_Anim_001674, &object_kbt_Anim_002084, &object_kbt_Anim_000FE8,
-        &object_kbt_Anim_001940, &object_kbt_Anim_00E7BC, &object_kbt_Anim_00F0C8, &object_kbt_Anim_002710,
-        &object_kbt_Anim_002DE0, &object_kbt_Anim_003414, &object_kbt_Anim_003D24, &object_kbt_Anim_001BF4,
-        &object_kbt_Anim_002084,
+void EnKbt_ChangeAnim(EnKbt* this, s16 animIndex) {
+    static AnimationHeader* sAnimations[ENKBT_ANIM_MAX] = {
+        &object_kbt_Anim_000670, // ENKBT_ANIM_0
+        &object_kbt_Anim_001674, // ENKBT_ANIM_1
+        &object_kbt_Anim_002084, // ENKBT_ANIM_2
+        &object_kbt_Anim_000FE8, // ENKBT_ANIM_3
+        &object_kbt_Anim_001940, // ENKBT_ANIM_4
+        &object_kbt_Anim_00E7BC, // ENKBT_ANIM_5
+        &object_kbt_Anim_00F0C8, // ENKBT_ANIM_6
+        &object_kbt_Anim_002710, // ENKBT_ANIM_7
+        &object_kbt_Anim_002DE0, // ENKBT_ANIM_8
+        &object_kbt_Anim_003414, // ENKBT_ANIM_9
+        &object_kbt_Anim_003D24, // ENKBT_ANIM_10
+        &object_kbt_Anim_001BF4, // ENKBT_ANIM_11
+        &object_kbt_Anim_002084, // ENKBT_ANIM_12
     };
-    u8 sp38[] = {
-        ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_LOOP, ANIMMODE_LOOP,
-        ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_ONCE,
+    u8 animationModes[ENKBT_ANIM_MAX] = {
```

Comment:
This one *isn't* `static`, but the previous one is. Is that required to match? Or can we make them consistent?

Reply:
No, it has to be this way unfortunately. `sAnimations` needs to be in-function and static, and `animationModes` needs to be in-function and non-static

Reply:
Ah, oh well. Thanks for checking!